_posts/2017/2017-08-17-tracing-the-suspect.md

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: "Tracing the suspect - a microservices murder mystery"
 date: "2017-08-17 09:30:00"
-image: blog-header/tracing-the-suspect.jpg
+image: blog-header/tracing-the-suspect-a-microservices-murder-mystery.jpg
```

Comment:
sorry, that was a misunderstanging. I was not relating to the image but to the .md file name